connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -329,9 +344,24 @@ private void rewind() {
         context.clearOffsets();
     }
 
+    protected void abort(Throwable t) {
+        // Shutdown the task now and close resources. This should only be called from the worker's task thread
```

Comment:
Nit: This class is an internal implementation, but shouldn't this be a javadoc on the class itself? "This should only be called from" is something that should be made obvious to users of the class.